app/models/test/client_request.rb

```diff
@@ -0,0 +1,23 @@
+module Test
```

Comment:
what is this about? If it is for text env, should we put it in `spec/support/fixtures` ?